demisto_sdk/commands/doc_reviewer/rn_checker.py

```diff
@@ -58,7 +58,9 @@ class ReleaseNotesChecker:
         'You can now',
         'Deprecated. ',
         'Deprecated the ',
-        'Note: '
+        'Note: ',
+        'Started adoption process',
+        'Completed adoption process',
```

Comment:
are these prefixes, or whole messages? If the latter, please add a `.` at the end.